README.md

```diff
@@ -244,15 +244,23 @@ development then sign up for an account of your own at
 
 Vagrant
 -------
-If you have Vagrant installed, you can run Gratipay by running `vagrant up` from the project directory. Please note that if you ever switch between running Gratipay on your own machine to Vagrant or vice versa, you will need to run `make clean`.
+Vagrant provides a convenient interface to VirtualBox to run and test
+Gratipay in virtual machine. This may be handy if you're on Windows.
 
-If you're using Vagrant for the first time you'll need [Vagrant](http://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/) installed. If you're on Linux you'll need to install `nfs-kernel-server`.
+You will [Vagrant](http://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/)
+installed. On Linux you may need to install `nfs-kernel-server` as well.
 
-The `Vagrantfile` will download a custom made image from the Internet. If you have a slow internet connection, you can download a local copy of this file, by running:
+With Vagrant, you can run Gratipay by running `vagrant up` from the project
+directory. Please note that if you switch between running Gratipay on your
+own machine to Vagrant and vice versa, you need to run `make clean`.
```

Comment:
Sometimes it happens. Fixed. =) At first I thought that the English was too complicated, but from the other side - how would people learn?